.pipelines/multitenancy/swiftv2-e2e-job-template.yaml

```diff
@@ -10,6 +10,7 @@ parameters:
 
 stages:
   - stage: ${{ parameters.clusterName }}
+    condition: and( succeeded(), not(eq(dependencies.dualstackoverlaye2e.result,'SucceededWithIssues')) ) # Cant use parameters in dependencies
```

Comment:
This cluster create job needs a continue on error flag, not a conditional. See https://github.com/Azure/azure-container-networking/blob/4370e6e93c7623a70446580417df1ab0fc2b4ff9/.pipelines/templates/create-cluster.yaml#L46